app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -617,15 +621,20 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope(), V
         }
     }
 
-    private fun notifyVpnStart() {
+    /**
+     * @return `true` if the start was initiated correctly, otherwise `false`
+     */
+    private fun notifyVpnStart(): Boolean {
         val vpnNotification =
             vpnEnabledNotificationContentPluginPoint.getHighestPriorityPlugin()?.getInitialContent()
-                ?: VpnEnabledNotificationContentPlugin.VpnEnabledNotificationContent.EMPTY
+                ?: return false
```

Comment:
Is there a realistic scenario when this could happen?
`vpnEnabledNotificationContentPluginPoint.getHighestPriorityPlugin` evaluates `isActive()` as well.

Reply:
Note: clarified offline